client/src/pages/Settings/EditProfile/EditProfile.tsx

```diff
@@ -98,7 +98,7 @@ const EditProfile: React.FC<EditProfileProps> = ({ header, currentUser, currentP
       <SettingHeader header={header} />
       <Formik
         initialValues={{
-          name: currentUser?.name || '',
+          name: currentProfile?.name || '',
```

Comment:
Form now uses `currentProfile`. Changes will be reflected on the `Profile` schema or it's inherited schemas.